applications/nrf_desktop/CMakeLists.txt

```diff
@@ -57,7 +57,7 @@ if (NOT CONFIG_NCS_IS_VARIANT_IMAGE)
     assert_exists(mcuboot_CONF_FILE)
   endif()
   if(CONFIG_SECURE_BOOT)
-    assert_exists(b0_CONF_FILE)
+    assert_exists(CONF_FILE)
```

Comment:
we should assert that configuration file for b0 exists - I doubt that this change is correct as it asserts application configuration file exists